github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml

```diff
@@ -437,6 +437,10 @@ orgs:
         description: Containerized Data Importer API definitions
         default_branch: main
         has_projects: true
+      application-aware-quota-api:
+        description: Application Aware Quota API definition
+        default_branch: main
+        has_projects: true
```

Comment:
Does this require github projects?

Reply:
TBH i'm not really sure but i saw that kubevirt/kubevirt and kubevirt/containerized-data-importer-api require projects as well.

Reply:
But now i see that they are not used :)
i will drop this.